### ISSUE TITLE
LC0005 False positive on DotNet datatypes (N.B. Allow aliases)

### ISSUE THREAD

**Issue by rvanbekkum** [StefanMaron/BusinessCentral.LinterCop#95]
Hi @StefanMaron,
I reported this one earlier to you via a DM, but I think you might have missed it in the final release.
The following variable/parameter declaration gives a false positive:
```
var vXMLNode: DotNet XmlNode
```
The alias is `XmlNode` which should be allowed, and is preferred.

It would also be fine for me to skip DotNet variables completely though.

**Comment by StefanMaron** [comment]
I have seen it, and I thought I adressed it.
![image](https://user-images.githubusercontent.com/24838311/141975303-10abc142-f32a-4329-862a-29377d6f286f.png)
![image](https://user-images.githubusercontent.com/24838311/141975328-d9d94d80-d48b-439e-837f-9c0d1e16f2d0.png)

Can you explain a bit more how to reproduce it?

**Comment by rvanbekkum** [comment]
The following example reproduces the issue:
```al
    procedure MyProcedure2(var myXMLNode: DotNet XmlNode)
    var
        myText: Text;
    begin
        myText := myXMLNode.BaseURI;
    end;
```

Seems to be only for parameters.
At the same time LC0003 raises a false positive here as well.

**Comment by StefanMaron** [comment]
On that snipped I only get a false positive for rule LC0003

**Comment by rvanbekkum** [comment]
I get false positives for both LC0003 and LC0005 😉.

![image](https://user-images.githubusercontent.com/7383241/141996541-09ebf21a-d2ec-477f-be14-e6fec263da44.png)

**Comment by rvanbekkum** [comment]
LC0003 and LC0005 now simply skip the subtype. (See #119)
